glide.lock

```diff
@@ -64,8 +64,6 @@ imports:
   subpackages:
   - rpcvmx
   - vmcheck
-- name: github.com/spf13/cobra
```

Comment:
Don't change this file manually, there is a digest at the top covering its content.